src/evm/prompts/implement-bls12-381-map-fp2-to-g2-precompile.md

```
@@ -0,0 +1,2451 @@
+// ... other precompile analysis functions
+
+PrecompileAnalysis bls12_map_fp2_to_g2_analyze(bytes_view, evmc_revision) noexcept
+{
+    // The gas cost is fixed as per the EIP specification.
+    static constexpr auto BLS12_MAP_FP2_TO_G2_PRECOMPILE_GAS = 23800;
+    return {BLS12_MAP_FP2_TO_G2_PRECOMPILE_GAS, BLS12_G2_POINT_SIZE};
+}
+
+// ... other precompile execution functions
+
+ExecutionResult bls12_map_fp2_to_g2_execute(const uint8_t* input, size_t input_size,
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers
null

(MD037, no-space-in-emphasis)